tests/listsAndDropDowns.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Lists and Dropdowns', async () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/')
+		await page.getByText('Owners').click()
+		await page.getByText('Search').click()
+		await expect(page.getByRole('heading')).toHaveText('Owners')
+	})
+
+	test('Validate selected pet types from the list', async ({ page }) => {
+		const ownerFullName = 'George Franklin'
+
+		await page.getByText(ownerFullName).click()
+		await expect(page.locator('.ownerFullName')).toHaveText(ownerFullName)
+
+		await page.getByText('Edit pet').click()
```

Comment:
If this owner would have two pets, or when second pet will be added for this owner - this test will fail